core/src/main/resources/oracle/weblogic/deploy/aliases/category_modules/SecurityConfiguration.json

```diff
@@ -1418,8 +1418,10 @@
         "RealmBootStrapVersion":                 [ {"version": "[10,12.2.1)", "wlst_mode": "offline", "wlst_name": "RealmBootStrapVersion",              "wlst_path": "WP001", "value": {"default": "None"    }, "wlst_type": "string",         "get_method": "GET", "restart_required": "true" } ,
                                                    {"version": "[10,)",       "wlst_mode": "online", "wlst_name": "RealmBootStrapVersion",               "wlst_path": "WP001", "value": {"default": "1"       }, "wlst_type": "string",         "get_method": "GET", "restart_required": "true" } ],
         "RemoteAnonymousJndiEnabled":            [ {"version": "[12.2.1.3,)", "wlst_mode": "both", "wlst_name": "RemoteAnonymous${Jndi:JNDI}Enabled",    "wlst_path": "WP001", "value": {"default": "true"    }, "wlst_type": "boolean",        "restart_required": "true"  } ],
-        "RemoteAnonymousRmiiiopEnabled":         [ {"version": "[14.1.1,)",   "wlst_mode": "both", "wlst_name": "RemoteAnonymous${Rmiiiop:RMIIIOP}Enabled", "wlst_path": "WP001", "value": {"default": "true" }, "wlst_type": "boolean"         } ],
-        "RemoteAnonymousRmit3Enabled":           [ {"version": "[14.1.1,)",   "wlst_mode": "both", "wlst_name": "RemoteAnonymous${Rmit3:RMIT3}Enabled",  "wlst_path": "WP001", "value": {"default": "true"    }, "wlst_type": "boolean"         } ],
+        "RemoteAnonymousRmiiiopEnabled":         [ {"version": "[12.2.1.4.10930,12.2.1.5)",   "wlst_mode": "online", "wlst_name": "RemoteAnonymous${Rmiiiop:RMIIIOP}Enabled", "wlst_path": "WP001", "value": {"default": "true" }, "wlst_type": "boolean"         },
```

Comment:
This is not the full list of changed attributes, nor does it include the versions for 12.2.1.3 and 12.2.1.5

Reply:
Backports were done as part of bug 31498405.  These two attributes are included in these two PSUs (and all future PSUs in the same branch)
PSU 12.2.1.3.210329
PSU 12.2.1.4.210330